Entity/User.php

```diff
@@ -199,4 +199,16 @@ public function isEnabled();
      * @return string
      */
     public function getUsername();
+
+
+    /**
+     * @return array
+     */
+    public function getRoles();
```

Comment:
```suggestion
    public function getRoles(): array;
```
Is this notation not supported for entities?